pages/dashboard/dashboard-page.js

```diff
@@ -27,6 +27,7 @@ exports.DashboardPage = class DashboardPage extends BasePage {
       'div[class*="dashboard-grid"] button[class*="create-new"]',
     );
     this.createFileButtonTitlePanel = page.getByTestId('project-new-file');
+    this.createFileButtonDraftsTab = page.getByTestId('project-new-file');
```

Comment:
I think this is not the locator you want to use, at least used from the test 1873.
As this test click on the Drafts projects before (`dashboardPage.openSidebarItem('Drafts')`), this locator won't never be visible as it never have a "+" button. 

Would it be correct to use `data-testid="new-file"` instead?
![data-testid](https://github.com/user-attachments/assets/da716409-64ed-49e1-9a7c-a04c8c2c07f4)